### PR TITLE
Fgts 191 add enemies to nasa hq

### PR DESCRIPTION
**Pull request by fheysel**
Adds all the enemies to the nasa level, also creates the vat of acid enemy

### CODE REVIEW

File: godotproject/Objects/Sawblade/SawBladeArea.tscn [hunk 1]
```diff
@@ -20,8 +20,8 @@ script = ExtResource( 4 )
 
 [node name="SawBlade" type="KinematicBody2D" parent="."]
 z_index = -1
-collision_layer = 64
-collision_mask = 0
+collision_layer = 68
+collision_mask = 4
```
Comment (jDenyes) [inline godotproject/Objects/Sawblade/SawBladeArea.tscn:24]: What collision layer is this adding?

Comment (fheysel) [reply godotproject/Objects/Sawblade/SawBladeArea.tscn:24]: we talked about this in standup but for record: adds enemy collision layer so rockets will explode on collision

File: godotproject/RocketEnemy.gd [hunk 2]
```diff
@@ -62,18 +62,26 @@ func _physics_process(delta):
 func _on_Area2D_body_entered(body):
 	if(body.name) == "Player":
```
Comment (charasyn) [inline godotproject/RocketEnemy.gd]:
```suggestion
	if Global.is_player(body):
```

File: godotproject/VatOfAcid.gd [hunk 3]
```diff
@@ -0,0 +1,6 @@
+extends Area2D
+
+
+func _on_VatOfAcid_body_entered(body):
+		if Global.is_player(body):
+			body.hit_death_plane()
```
Comment (charasyn) [inline godotproject/VatOfAcid.gd:6]: 👍

**Review by charasyn — APPROVED** [review]
Looks good to me, just the one change to use the Global.is_player function

**Pull request merged**